cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts

```diff
@@ -71,6 +71,9 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
 
         const tabbedOptions = new TabbedPo();
 
+        // Latest (`104.0.0+up45.31.1`) is broken (crd installs, but not actual chart), use `103.1.1+up45.31.1` instead
+        chartPage.selectVersion('103.1.1+up45.31...');
```

Comment:
We could probably leave off the ... - I assume that's to do with how we present it?

Reply:
correct. i tried the full one first but that fails.

removing the ... would be better. will do it if there's another round